src/components/StatisticalChart/StatisticalChartReducer.js

```diff
@@ -16,9 +16,7 @@ export const StatisticalChartReducer = (state = initialState, action) => {
       return {
```

Comment:
I believe you have not applied your recent changes to `initialState` as well. Please consider fixing that.

Reply:
Also, I think if you don't handle the `error` scenario for this request, the application will crash due to further assumptions made about the `data` variable. (https://github.com/mask-application/pwa/blob/HomePageTopChart/src/components/StatisticalChart/StatisticalChart.js#L13, https://github.com/mask-application/pwa/blob/HomePageTopChart/src/components/StatisticalChart/StatisticalChartGuide.js#L7)